github/github.go

```diff
@@ -301,18 +303,31 @@ func addOptions(s string, opts interface{}) (string, error) {
 	return u.String(), nil
 }
 
+type clientOption func(*Client)
+type secondaryRateLimitHook func(ctx context.Context, client *Client, req *http.Request, orgError *AbuseRateLimitError) (*Response, error)
+
+func WithSecondaryRateLimitHook(hook secondaryRateLimitHook) clientOption {
```

Comment:
Please add GoDoc-style comments (complete sentences ending in punctuation) to the 3 new exported items on lines 306, 307, and 309.